README.md

```diff
@@ -1,147 +1,156 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 Personal portfoilo <a name="about-project"></a>
+
+
+**Personal Portfolio** Is my personal website built to give my future employers a look at my work and series of projects listed in it 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- code editor(preferably vscode)
+- Git Bash
+- Brower
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Danny-imo/portfolio.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd portfolio
+   npm install --save-dev hint@7.x
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npx hint .
+  npx stylelint "**/*.{css,scss}"
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Imo Daniel**
+
+- GitHub: [@Danny-imo](https://github.com/Danny-imo)
+- Twitter: [@danielimo758](https://twitter.com/danielimo758)
+- LinkedIn: [@imo-daniel](https://www.linkedin.com/in/imo-daniel-b15585167)
+- Facebook: [@imoh-daniel](https://www.facebook.com/imoh.daniel.399?mibextid=ZbWKwL)
+
+
+
+👤 **Lala El vadel**
+
+- GitHub: [@leloufadel](https://github.com/leloufadel)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Key Features <a name="key-features"></a>
+
+
+- [ ] **HTML**
+- [ ] **CSS**
+- [ ] **Linters**
+- [ ] **Animations**
+- [ ] **Device compatible responsiveness**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://danny-imo.github.io/portfolio/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **JavaScript**
+- [ ] **Dynamics**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank my partners
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+
+## 📝 License <a name="license"></a>
 This project is [MIT](https://choosealicense.com/licenses/mit/) licensed .
```

Comment:
- [ ] The link to the LICENSE displays the template sample of the MIT LICENSE and not your personalized sample. Kindly create a valid LICENSE for this project by following the highlighted instructions on the screenshot attached below 🔽 :

![Screenshot_2](https://user-images.githubusercontent.com/77106962/230966509-07d2f7f5-e4f1-4c06-a8ea-d5df777240ed.png)